javascript/src/main/components/BasicCourseSearch/BasicCourseTable.js

```diff
@@ -200,6 +200,14 @@ const BasicCourseTable = ({ classes, checks, displayQuarter, allowExport }) => {
       text: 'YYYYQ'
     }
   );
+
+  if ((sections.length === 0) || (sections===null)) {
+    return (
+      <div data-testid="no-course-data">
+      </div>
+    )
+  }
+  
```

Comment:
We should really make both a test for this case, and a storybook for this case.

Reply:
Oh, nvm about the story part... that's already done!  Let's just focus on the testing part.